lib/trailblazer/operation.rb

```diff
@@ -29,8 +29,8 @@ module Trailblazer
   class Operation
     module FastTrackActivity
       builder_options = {
-        track_end:     Railway::End::Success.new(semantic: :success),
-        failure_end:   Railway::End::Failure.new(semantic: :failure),
+        track_end: Railway::End::Success.new(semantic: :success),
```

Comment:
can you revert these style changes ?

Reply:
@seuros they are demanded by ` .rubocop-https---raw-githubusercontent-com-trailblazer-meta-master-rubocop-yml` - I didn't add it and I guess you guys added that to the projects config?

Reply:
@seuros